client/src/pages/Paste.js

```diff
@@ -38,6 +65,14 @@ function Paste({ match }) {
           </CreatedAt>
           <Author name={paste.author} />
           <PasteArea>{paste.value}</PasteArea>
+          <EmailInput
+            placeholder="Enter the email of your recipient"
+            value={inputValue}
+            type="email"
+            onChange={event => setInputValue(event.target.value)}
+          ></EmailInput>
+          <SendEmailButton onClick={handleClickEvent}></SendEmailButton>
+          <div>{Approval(approval)}</div>
```

Comment:
The email type check does not work. We were able to get a "Email successfully send" without a valid email entry (i.e. "test@"). 

Mayber include an additional check for correct email syntax.

Reply:
Yeah, we are currently working on that problem. Thx :)

Reply:
I fixed the email validation with adding a form element.  You'll still get an  "Email successfully send"  because for now it's just an alert.